### DISCUSSION TITLE
可否支持全局配置 select  optionLabelProp="label"

### DISCUSSION THREAD

**Discussion by vaynevayne** [ant-design/ant-design#46118] [Q&A]
如题

**Comment by afc163** [comment]
建议封装一个业务自己用的 Select。

**Reply by vaynevayne** [reply]
config 里支持 showSearch, 可以再添加一个optionFilterProp吗?

**Reply by vaynevayne** [reply]
https://github.com/ant-design/ant-design/pull/46119 透传所有的属性 应该没问题吧
